bundles/org.openhab.binding.tado/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -171,7 +172,7 @@
 		<label>Target Temperature</label>
 		<description>Thermostat temperature setpoint</description>
 		<category>Temperature</category>
-		<state step="0.1" pattern="%.1f %unit%" readOnly="false"></state>
+		<state step="0.1" pattern="%.1f %unit%" readOnly="false"/>
```

Comment:
Since you are already simlifying, you could also leave the default here:
```suggestion
		<state step="0.1" pattern="%.1f %unit%"/>
```

Reply:
Ok.